src/components/Divider/stories/Divider.story.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { SafeAreaView, View, Text, StyleSheet } from 'react-native';
+import { storiesOf } from '@storybook/react-native';
+
+import { colors, sizes, typography } from '@magnetis/astro-tokens';
+import { select } from '@storybook/addon-knobs';
+
+import Divider from '../Divider';
+import { DividerOrientationOptions, DividerSize, DividerSizeOptions, DividerTypeOptions } from '../types';
```

Comment:
❓ Aqui precisaria adicionar o `type`?
```suggestion
import type { DividerOrientationOptions, DividerSize, DividerSizeOptions, DividerTypeOptions } from '../types';
```

Reply:
Acredito que eu poderia dar uma ajustada, porque Os `...Options` não são types, mas arrays criados através de types que existem nesse arquivo. Já o `DividerSize` é um type mesmo.

Reply:
Fixed in [61de2ac](https://github.com/magnetis/astro-native/pull/782/commits/61de2acb8a0d6e0340f134e916e88bc4e2aa5242)

Reply:
I don't see the contants.ts file